src/tdlib/chat.rs

```diff
@@ -412,6 +412,10 @@ impl Chat {
         self.type_().user() == Some(&self.session().me())
     }
 
+    pub(crate) fn is_replies_chat(&self) -> bool {
+        self.id() == 1271266957
```

Comment:
Note: It does it's job, but it would be more clear to compare chat against `replies` username. Not sure if telegram might change the id of replies chat, but if they ever will it will break your changes. But on the other hand comparing usernames requires more computations

Reply:
imho it's easier for the replies chat username to get replaced than the id. Telegram Desktop also does the comparison by id:
https://github.com/telegramdesktop/tdesktop/blob/41d9a9fcbd0c809c60ddbd9350791b1436aff7d9/Telegram/SourceFiles/data/data_peer.cpp#L891

And now turns out that the replies chat has a different id in the test backend, I should probably add support for it too.